src/scripts/breeding/HatcheryHelper.ts

```diff
@@ -75,13 +75,23 @@ class HatcheryHelper {
         return this.unlockRequirement?.isCompleted() ?? true;
     }
 
+    // String for currency in Notifications and Logs
+    currencyString(string: string): any {
+        if (GameConstants.humanifyString(GameConstants.Currency[this.cost.currency]) != 'money') {
+            string = (`${GameConstants.camelCaseToString(GameConstants.Currency[this.cost.currency])}s`);
+        } else {
+            string = ('Pokédollars');
+        }
+        return string;
+    }
```

Comment:
Why does this have a string input, it looks like it will always be overwritten?
Also this feels like it should be a switch-case instead, but I assume eslint complains if we do that without enough cases...

Reply:
Does the change I just pushed look better? Doesn't look like any eslint complaints about the single switch case.